### DISCUSSION TITLE
Open Tablet Driver  found another driver i cant find it tho

### DISCUSSION THREAD

**Discussion by WitchofGreed** [OpenTabletDriver/OpenTabletDriver#2741] [Q&A] [answered]
Open Tablet Driver found a driver from gaomon which i cant seem to find anywhere. I deinstalled every gaomon driver i had before and i installed Open Tablet Driver now i dont know what to do.

**Comment by jamesbt365** [comment, marked as answer]
https://github.com/X9VoiD/TabletDriverCleanup/releases/latest

**Reply by WitchofGreed** [reply]
That worked thanks, but now the cursor doesnt move if i use the pen...

**Reply by WitchofGreed** [reply]
okay nvm it works now thx alot
